source/a_release_25062019.rst

```diff
@@ -201,31 +202,31 @@ By clicking on one of the classes you will find the regions where that specific
 * Bug in selecting a line or polygon asset (reference: PROJ-782)
 
 Lizard Management Interface (demo.lizard.net/management/)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+---------------------------------------------------------
 
 **User Management**
 
 We’ve developed a user friendly user management interface. With this interface managers can add users to their organisation and give them the right authorization to data and applications. 
```

Comment:
authoriZation: z of s? Volgens mij zag ik elders *the queen's English*, dus zou een S beter zijn?
Eventueel even een beetje greppen (wat ook later kan).

Reply:
Deze zit er nog in. Algemene search/replace doen?

Reply:
Fixed, komt met de volgende push,